tests/puppeteer/pages/commonPage.js

```diff
@@ -253,4 +256,21 @@ module.exports = class CommonPage {
       await this.page.click(checkboxSelector);
     }
   }
+
+  /**
+   * Check File existence
+   * @param downloadPath
+   * @param fileName
+   * @param timeDelay
+   * @return boolean, true if exist, false if not
+   */
+  async checkFileExistence(downloadPath, fileName, timeDelay = 5000) {
+    for (let i = 0; i < timeDelay; i++) {
+      await this.page.waitFor(1);
+      if (fs.existsSync(`${downloadPath}/${fileName}`)) {
+        break;
+      }
+    }
+    return fs.existsSync(`${downloadPath}/${fileName}`);
+  }
```

Comment:
```js
  async checkDownloadedFile(downloadPath, fileName, timeDelay = 5000) {
    let found = false;
    for (let i = 0; i < timeDelay && !found; i += 10) {
      setTimeout(() => { /* only to sleep */ }, 10);
      found = await this.isFileExist(downloadPath, fileName);
    }
    return found;
  }
```
- Maybe you should reduce frequency of testing if file exist (to 10 or 20 ms not 1)
- Maybe you should use utils/files.js and not the common page
- Maybe it's better to use var found to not check the existence of the file again while returning value.
(to be tested)